container-storage-setup.sh

```diff
@@ -888,6 +876,38 @@ create_disk_partitions() {
   done
 }
 
+remove_partition() {
+  local dev="$1"
+
+  if [ -x "/usr/sbin/parted" ]; then
+    parted "$dev" rm 1
+  else
+    sfdisk --delete "$dev" 1
+  fi
+}
+
+# Remove disk pvs and partitions. This is called in reset storage path.
+# If partition or pv does not exist, it will still return success. Error
+# willl be returned only if pv or partition existst and removal fails.
```

Comment:
+# willl be returned only if pv or partition existst and removal fails.
SPELLING =========================^